Foundation/NSStream.swift

```diff
@@ -109,24 +107,71 @@ public class Stream: NSObject {
     }
     
     /*@NSCopying */public var streamError: NSError? {
-        NSUnimplemented()
+        NSRequiresConcreteImplementation()
     }
+    
+}
+
+internal final class _SwiftInputStream : NSInputStream, _SwiftNativeFoundationType {
+    internal typealias ImmutableType = NSInputStream
+    internal typealias MutableType = NSInputStream
```

Comment:
not sure about setting MutableType to NSInputStream but only way to satisfy _SwiftNativeFoundationType's constraints